deploy/helm/templates/deployment.yaml

```diff
@@ -1,39 +1,26 @@
-apiVersion: v1
-kind: Service
-metadata:
-  labels:
-    app: gitlint-bot-production
-  name: gitlint-bot-production
-  namespace: gitlint-bot
-spec:
-  ports:
-    - port: 3000
-      protocol: TCP
-      targetPort: 3000
-  selector:
-    app: gitlint-bot-production
-  type: ClusterIP
----
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app: gitlint-bot-production
-  name: gitlint-bot-production
-  namespace: gitlint-bot
+    app: "{{ .Values.appName }}"
+    version: "{{ .Values.appVersion }}"
+    {{- include "gitlint-bot.labels" . | nindent 4 }}
+  name: "{{ .Values.appName }}"
+  namespace: "{{ .Values.namespace }}"
 spec:
-  replicas: 1
+  replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      app: gitlint-bot-production
+      app: "{{ .Values.appName }}"
   template:
     metadata:
       labels:
-        app: gitlint-bot-production
+        app: "{{ .Values.appName }}"
+        version: "{{ .Values.appVersion }}"
     spec:
       containers:
-        - name: gitlint-bot-production
-          image: oursky/gitlint-bot:latest
+        - name: "{{ .Values.appName }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.appVersion | default "latest" }}"
```

Comment:
We should be using `.Values.image.tag` for image tag by convention. There's no need to give a default value - chart values are default values.